x-pack/packages/kbn-elastic-assistant/impl/assistant/conversations/conversation_settings/conversation_settings.tsx

```diff
@@ -415,6 +426,21 @@ export const ConversationSettings: React.FC<ConversationSettingsProps> = React.m
               />
             </EuiFormRow>
           )}
+        <EuiSpacer size="l" />
+        <EuiTitle size={'s'}>
+          <h2>{i18n.SETTINGS_ALL_TITLE}</h2>
+        </EuiTitle>
+        <EuiSpacer size="xs" />
+        <EuiText size={'s'}>{i18n.SETTINGS_ALL_DESCRIPTION}</EuiText>
+        <EuiHorizontalRule margin={'s'} />
+        <EuiFormRow display="rowCompressed" label={i18n.STREAMING_TITLE}>
```

Comment:
nit: add `fullWidth` to match wrapping of other settings components

```suggestion
        <EuiFormRow display="rowCompressed" fullWidth label={i18n.STREAMING_TITLE}>
```

<p align="center">
  <img width="500" src="https://github.com/elastic/kibana/assets/2946766/c48aed00-d99b-457f-9808-b4bf07ff9494" />
</p>